src/tests/Microsoft.PowerFx.Core.Tests/Microsoft.PowerFx.Core.Tests.csproj

```diff
@@ -63,6 +63,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <None Remove="ExpressionTestCases\If_AllowsSideEffects.txt" />
     <None Remove="ExpressionTestCases\SqrtT.txt" />
   </ItemGroup>
```

Comment:
can we remove? VS may have added accidentaly

Reply:
Okay. Should I remove both?

Reply:
Done.